region/norfair/crocomire/Grapple Tutorial Room 3.json

```diff
@@ -111,13 +112,81 @@
         }
       }
     },
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway - Frozen Gamet",
+      "requires": [
+        "h_canNavigateUnderwater",
+        {"or": [
+          "h_canTrickyFrozenEnemyRunway",
+          {"and": [
+            "h_canFrozenEnemyRunway",
+            "Morph"
+          ]}
+        ]}
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 6,
+          "openEnd": 0
+        }
+      },
+      "note": [
+        "Lure the Geemers up out of the water. It may be helpful to freeze them on the ascent to prevent them from separating and moving horizontally.",
+        "At the top, freeze a Gamet while there is a half-tile gap between it and the runway in order to extend it as much as possible.",
+        "Positioning is much easier with Morph; simply morph on the runway before they start separating.",
+        "Freezing one or two Gamets after they separate but before they start moving can help with dodging and avoiding damage."
+      ]
+    },
     {
       "link": [1, 1],
       "name": "Gamet Farm",
       "requires": [
         {"refill": ["Energy", "Missile", "Super", "PowerBomb"]}
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Leave With Runway - Frozen Gamet",
+      "requires": [
+        "h_canNavigateUnderwater",
+        "canTrickyJump",
+        {"or": [
+          "h_canTrickyFrozenEnemyRunway",
+          {"and": [
+            "h_canFrozenEnemyRunway",
+            "Morph"
+          ]}
+        ]},
+        {"or": [
+          "SpaceJump",
+          {"and": [
+            "Gravity",
+            "canWalljump"
+          ]}
+        ]},
+        {"or": [
+          {"ammo": {"type": "Super", "count": 1}},
+          {"obstaclesCleared": ["A"]}
+        ]}
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 9,
+          "openEnd": 0
+        }
+      },
+      "unlocksDoors": [
+        {"types": ["ammo"], "requires": []}
+      ],
+      "note": [
+        "Lure the Geemers up out of the water. It may be helpful to freeze them on the ascent to prevent them from separating and moving horizontally.",
+        "At the top, position a Gamet flush with the central runway. Positioning is much easier with Morph; simply morph on the runway before they start separating.",
+        "Freezing one or two Gamets after they separate but before they start moving can help with dodging and avoiding damage.",
+        "Lure the Gamet to the right and freeze it while there is a half-tile gap between it and the runway in order to extend it as much as possible."
+      ],
+      "devNote": "This can be done without Gravity or Space Jump, but it is quite a bit harder and would be too much to extend a runway by a tile."
```

Comment:
Ice only looked very reasonable to me.  
Freeze one at the right height, from the left stairs, once the 5 split apart and kill any that are in the way.  Use it as a platform to reach mid.  Then it will unfreeze and turn right.  Jump ahead of it and land on it in the middle of the water (easier with morph).  Then jump the rest of the way.

Without Morph I just did a shot into jump, so they would rise a little more as the shot was travelling.  But that's a lot more reliable with gravity.  Then jumped into the water, so I was lower, and refroze once they split.  Used the background tiling to check pixel height.

Reply:
This was quite hard for me at first, but now it doesnt feel bad